client/src/components/MultiJurisdictionAudit/RoundManagement/_mocks.ts

```diff
@@ -0,0 +1,95 @@
+import { IAuditBoard } from '../useAuditBoards'
+
+export const auditBoardMocks: {
```

Comment:
could these mocks be combined with the mocks in `MultiJurisdictionAudit/_mocks`? I have been trying to follow your pattern of consolidating mocks there so we can reuse the same set more often